src/olympia/addons/tests/test_models.py

```diff
@@ -1704,26 +1704,35 @@ def setUp(self):
         self.addon = Addon.objects.get(id=3615)
         self.version = self.addon.current_version
 
-    def new_version(self, status):
-        version = Version.objects.create(addon=self.addon)
-        File.objects.create(version=version, status=status)
-        return version
-
     def test_public_new_nominated_version(self):
-        self.new_version(amo.STATUS_NOMINATED)
-        assert self.addon.find_latest_public_version() == self.version
+        version_factory(
+            addon=self.addon, file_kw={'status': amo.STATUS_NOMINATED})
```

Comment:
that's not a valid status for File (ugh for old tests).  Should only ever be one of `STATUS_CHOICES_FILE`